docs/pages/access-controls/sso/okta.mdx

```diff
@@ -114,13 +138,15 @@ Create the connector using `tctl` tool:
 $ tctl create okta-connector.yaml
 ```
 
-## Create a Developer Teleport Role
+## Step 4/4. Create a developer Teleport role
 
-We are going to create a new role that'll pull in external information from Okta. Notice
-`{{external.username}}` login. It configures Teleport to look at *"username"* Okta claim
-and use that field as an allowed login for each user.  This example uses email as the
-username format.  The `email.local(external.username)` function call will remove
-the `@domain` and leave the username prefix.
+Now let's create a new role to pull in external information from Okta.
+Create the local file `dev.yaml` with the content below.
+
+Notice the `{{external.username}}` login. It configures Teleport to look at the
```

Comment:
Should this paragraph go after the code snippet so the connection between "the content below" and the snippet to copy is more obvious?